src/ECharts/Theme.purs

```diff
@@ -1,58 +1,45 @@
 module ECharts.Theme
   ( Theme(..)
+  , BuiltInTheme
+  , builtInThemeName
   , dark
   , infographic
   , macarons
   , roma
   , shine
   , vintage
-  , BuiltInTheme(..)
-  , parseBuiltInTheme
-  , builtInToTheme
   ) where
 
-import Prelude (($), (<>))
 import Data.Either (Either(..))
 import Data.Foreign (Foreign)
 
-foreign import forceExport ∷ Foreign
+type Theme = Either BuiltInTheme Foreign
 
-data Theme = ByName String | FromObject Foreign
-data BuiltInTheme = Infographic | Macarons | Roma | Shine | Vintage | Dark
+foreign import data BuiltInTheme :: Type
 
-parseBuiltInTheme ∷ String → Either String BuiltInTheme
```

Comment:
> I wouldn't think dynamic theme selection from a string is the normal use case 

Good point, ps-echarts might not be best place for `builtInThemeFromName`, will move that to slamdata.

@cryogenian  do you agree?